src/System.ValueTuple/src/System.ValueTuple.builds

```diff
@@ -6,6 +6,9 @@
     <Project Include="System.ValueTuple.csproj">
       <TargetGroup>portable-net40+sl4+win8+wp8</TargetGroup>
     </Project>
+    <Project Include="System.ValueTuple.csproj">
```

Comment:
When passed in, `TargetGroup` is used to define `NugetTargetMoniker` and `PackageTargetFramework`

When `TargetGroup` is not passed in from a .builds file to a .csproj, then the default is the value of `<NugetTargetMoniker>` defined in the .csproj itself, which in this case is: `.NETStandard,Version=v1.0`